airbnb_project/listings/listing_models.py

```diff
@@ -1,16 +1,19 @@
 from django.db import models
 
 class Listing(models.Model):
-  name = models.CharField(max_length=100)
-  baths = models.CharField(max_length=10)
-  beds = models.CharField(max_length=10)
-  latitude = models.CharField(max_length=20)
-  location = models.CharField(max_length=20)
-  longitude = models.CharField(max_length=20)
-  person_capacity = models.CharField(max_length=20)
-  registration_number = models.CharField(max_length=20)
-  room_type = models.CharField(max_length=20)
-  title = models.CharField(max_length=100)
+    airbnb_listing_id = models.TextField(blank=False, null=False)
+    name = models.TextField(null=True, blank=True)
+    baths = models.TextField(null=True, blank=True)
+    beds = models.TextField(null=True, blank=True)
+    latitude = models.TextField(null=True, blank=True)
+    location = models.TextField(null=True, blank=True)
+    longitude = models.TextField(null=True, blank=True)
+    person_capacity = models.TextField(null=True, blank=True)
```

Comment:
Should this be a Number/Integer type instead? `models.IntegerField()`

Reply:
I put it as text because I am not sure if the what is returned from airbnb will always be parseable to a numberm, just to be safe.